src/main/kotlin/com/hibob/kotlinBascis/Meeting.kt

```diff
@@ -0,0 +1,39 @@
+package com.hibob.kotlinBascis
+
+abstract class Meeting(open val name:String, open val location:Location)
+
+class TeamMeeting(name: String, location: Location, participants: List<Participant>) : Meeting(name, location) {
+    private val participants: MutableList<Participant> = mutableListOf()
+
+    init {
```

Comment:
why do you need that?

Reply:
Hi, maybe I didn’t quite understand you… You told me earlier that I need to add a function to allow someone to join a meeting. So initially, the meeting can start with a certain number of people, and I thought it would be possible to add more people later… I would appreciate some guidance if that wasn’t the direction.